### PR TITLE
Update list of IRC channels

### PR DESCRIPTION
**Pull request by aklapper**
https://phabricator.wikimedia.org/T56230

### CODE REVIEW

File: browser/data/json/irc-repos.json [hunk 1]
```diff
@@ -1 +1 @@
-["wikimedia-operations", "mediawiki", "wikimedia-dev", "wikimedia-labs", "wikimedia-mobile", "wikimedia-toolserver", "wikimedia-tech", "wikimedia-analytics", "wikimedia-fundraising", "semantic-mediawiki", "huggle"]
\ No newline at end of file
+["mediawiki", "mediawiki-i18n", "semantic-mediawiki", "mediawiki-parsoid", "mediawiki-visualeditor", "mediawiki-scripts", "wikimedia-tech", "wikimedia-dev", "wikimedia-toolserver", "wikimedia-labs", "wikimedia-labs-nagios", "wikimedia-ai", "wikimedia-analytics", "wikimedia-cep", "mediawiki-core", "wikimedia-collaboration", "wikimedia-databases", "wikimedia-design", "wikimedia-ect", "wikimedia-fundraising", "wikimedia-mobile", "wikimedia-multimedia", "wikimedia-operations", "wikimedia-perf", "wikimedia-qa", "wikimedia-releng", "wikimedia-research", "wikimedia-search", "wikimedia-teampractices", "huggle", "pywikibot", "wikidata"]
```
Comment (nemobis) [inline browser/data/json/irc-repos.json]: #mediawiki-i18n is not publicly logged, does that break something?

**Comment by aklapper** [comment]
That is a very good question by nemobis above and I don't know the answer. Bitergia can hopefully answer this. I assume I should remove mediawiki-i18n from that list but not sure how to "test" whether more IRC channels are not publicly logged and would also be affected...

**Comment by dicortazar** [comment]
Hi there,

First, thanks a lot for this change!

The issue here is that the JSON files are the output of the toolchain and not something that later feeds some other tools.

In other words: that JSON file is the result of asking the database what IRC channels are available.

What I'll do is to add that list of channels to the list of repositories used by the Wikimedia Foundation in [1].

I'll check how many of them are already analyzed and I'll add the rest of them.

Regarding to the addition of channels not publicly logged, we have an issue: no logs, no metrics.



[1] https://github.com/Bitergia/mediawiki-repositories

**Comment by dicortazar** [comment]
I've updated the current list of analyzed IRC channels in https://github.com/Bitergia/mediawiki-repositories/commit/65961e5daf7f7a6e26b81c7ceacd14ee718f5f46

**Comment by dicortazar** [comment]
Moving discussion to https://phabricator.wikimedia.org/T56230. Closing this pull request.

Thanks for your time here!

**Pull request closed without merging**